packages/components/e2e/component-objects/pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.talend</groupId>
     <artifactId>component-objects</artifactId>
-    <version>1.0.0</version>
+    <version>0.95.0-SNAPSHOT</version>
```

Comment:
I discussed with @kmouzouri and @atriki from TFD team, they are the only ones using the lib, and they don't maintain any (release) branch for now. So it's the time to align version and override the 1.0.0.